Description: Disable support for PARTITION_DIAG, not yet implemented upstream
Origin: https://fedorahosted.org/pyparted/changeset/e3d63b886f430a90e58a92e70c6f11ba92c695e5

Index: pyparted/src/_pedmodule.c
===================================================================
--- pyparted.orig/src/_pedmodule.c	2010-05-01 12:12:07.298587760 +0200
+++ pyparted/src/_pedmodule.c	2010-05-01 12:11:09.642584617 +0200
@@ -482,7 +482,6 @@
     PyModule_AddIntConstant(m, "PARTITION_MSFT_RESERVED", PED_PARTITION_MSFT_RESERVED);
     PyModule_AddIntConstant(m, "PARTITION_APPLE_TV_RECOVERY", PED_PARTITION_APPLE_TV_RECOVERY);
     PyModule_AddIntConstant(m, "PARTITION_BIOS_GRUB", PED_PARTITION_BIOS_GRUB);
-    PyModule_AddIntConstant(m, "PARTITION_DIAG", PED_PARTITION_DIAG);
 
     PyModule_AddIntConstant(m, "DISK_CYLINDER_ALIGNMENT", PED_DISK_CYLINDER_ALIGNMENT);
 
Index: pyparted/src/parted/__init__.py
===================================================================
--- pyparted.orig/src/parted/__init__.py	2010-05-01 12:12:07.010587523 +0200
+++ pyparted/src/parted/__init__.py	2010-05-01 12:11:24.298581393 +0200
@@ -106,7 +106,6 @@
 from _ped import PARTITION_MSFT_RESERVED
 from _ped import PARTITION_APPLE_TV_RECOVERY
 from _ped import PARTITION_BIOS_GRUB
-from _ped import PARTITION_DIAG
 
 from _ped import DISK_CYLINDER_ALIGNMENT
 
Index: pyparted/tests/_ped/test_partition.py
===================================================================
--- pyparted.orig/tests/_ped/test_partition.py	2010-05-01 12:12:08.342591411 +0200
+++ pyparted/tests/_ped/test_partition.py	2010-05-01 12:11:55.046584808 +0200
@@ -110,7 +110,7 @@
                      _ped.PARTITION_HPSERVICE, _ped.PARTITION_PALO,
                      _ped.PARTITION_PREP, _ped.PARTITION_MSFT_RESERVED,
                      _ped.PARTITION_APPLE_TV_RECOVERY,
-                     _ped.PARTITION_BIOS_GRUB, _ped.PARTITION_DIAG]:
+                     _ped.PARTITION_BIOS_GRUB]:
             self.assertTrue(isinstance(self._part.is_flag_available(flag), bool))
 
         # However, an invalid flag should definitely not be available.
Index: pyparted/tests/_ped/test_ped.py
===================================================================
--- pyparted.orig/tests/_ped/test_ped.py	2010-05-01 12:12:08.046588265 +0200
+++ pyparted/tests/_ped/test_ped.py	2010-05-01 12:12:05.246582473 +0200
@@ -41,7 +41,7 @@
                   _ped.PARTITION_PALO, _ped.PARTITION_PREP,
                   _ped.PARTITION_MSFT_RESERVED,
                   _ped.PARTITION_APPLE_TV_RECOVERY,
-                  _ped.PARTITION_BIOS_GRUB, _ped.PARTITION_DIAG]:
+                  _ped.PARTITION_BIOS_GRUB]:
             self.assertNotEquals(_ped.partition_flag_get_name(f), "", "Could not get name for flag %s" % f)
 
         self.assertRaises(ValueError, _ped.partition_flag_get_name, -1)
